src/com/homepractice/InheritanceConcept.java

```diff
@@ -0,0 +1,31 @@
+package com.homepractice;
+
+public class InheritanceConcept {
+
+	public static void main(String[] args) {
+		C c = new C();
+		c.s="Priya";  
+		P p = new P();
+		p.add();
+		// p.sub(); cannot access child method by using parent ref
```

Comment:
why?

Reply:
if you create an object using parent reference..parent reference has only access to parent methods not to child method .because object is created for child only.